clang/test/CIR/CodeGen/static.cpp

```diff
@@ -31,7 +31,7 @@ static Init __ioinit2(false);
 // BEFORE-NEXT: }
 
 
-// AFTER:      module {{.*}} {
+// AFTER:      module {{.*}} attributes {{.*}}cir.globalCtors = [#cir.globalCtor<"__cxx_global_var_init", 65536>, #cir.globalCtor<"__cxx_global_var_init.1", 65536>]
```

Comment:
Nice!!! This can be a bit more terse if we don't print the priority when it's the default value. In tablegen, you could instead add priority as an optional param and use optional groups (https://mlir.llvm.org/docs/DefiningDialects/Operations/#optional-groups) in the assemblyFormat description. Then the semantic could change a bit: if the optional has no value in CIR, it means we don't care about the priority (and during LLVM lowering we set those as 65536). How about that?

Reply:
Sounds good. This is cleaner!